DocumentAi/src/V1/Document/Provenance/Parent.php

```diff
@@ -0,0 +1,149 @@
+<?php
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# source: google/cloud/documentai/v1/document.proto
+
+namespace Google\Cloud\DocumentAI\V1\Document\Provenance;
+
+use Google\Protobuf\Internal\GPBType;
+use Google\Protobuf\Internal\RepeatedField;
+use Google\Protobuf\Internal\GPBUtil;
+
+/**
+ * Structure for referencing parent provenances.  When an element replaces
+ * one of more other elements parent references identify the elements that
+ * are replaced.
+ *
+ * Generated from protobuf message <code>google.cloud.documentai.v1.Document.Provenance.Parent</code>
+ */
+class Parent extends \Google\Protobuf\Internal\Message
```

Comment:
This class may actually be invalid on PHP < 7.0, which is why the `PBParent` class was being generated before.

Reply:
It turns out yes, the tests are failing because this name is reserved.